test/files/pos/macro-annot-unused-param.flags

```diff
@@ -0,0 +1,3 @@
+-Ymacro-annotations
+-Wunused:params
+-Werror
```

Comment:
Avoid the extra file by a code comment in Test_2:
`// scalac: -Ymacro-annotations -Wunused:params -Werror`

Reply:
oh yeah, that's the way it's done in 2.13.x...

Reply:
Thanks, didn't know I could do that.

Reply:
I started removing flags files yesterday, it was soothing busy-work for a slow coronavirus day. I made a big PR of it.